### ISSUE TITLE
Add documentation for new Twint component 

### ISSUE THREAD

**Issue by ronaldvanduren** [Adyen/adyen-android#1707]
**Is your feature request related to a problem? Please describe.**
We are trying to integrate the new Twint component, however, there is no documentation. We tried to initialize it the same way as the Card component, which "works". This shows the component always in a "payment already in progress" state. But it's unclear how to initialize the component properly so it opens the Twint app on the device. 

**Describe the solution you'd like**
Documentation that describes in-depth how to use the component. 

**Describe alternatives you've considered**
We have looked at the documentation of the card component, which helped us somewhat.

**Comment by araratthehero** [comment]
Hi @ronaldvanduren, thanks for raising this issue.

We are currently working on adding documentation for Twint, but we don't have a release date yet.

In the meantime, I can help you set it up.

Could you please follow these steps:
- Follow the [Android Components integration guide](https://docs.adyen.com/online-payments/build-your-integration/sessions-flow?platform=Android&integration=Components).
- [Import](https://docs.adyen.com/online-payments/build-your-integration/sessions-flow/?platform=Android&integration=Components#import) the `instant` module.
- [Launch and show](https://docs.adyen.com/online-payments/build-your-integration/sessions-flow?platform=Android&integration=Components#launch-and-show) the InstantPaymentComponent.

If you want to change the action handling method for Twint (default value is `PREFER_NATIVE`), you need to:
- [Import](https://docs.adyen.com/online-payments/build-your-integration/sessions-flow/?platform=Android&integration=Components#import) the `twint` module.
- Set the action handling module in your configuration
```
CheckoutConfiguration(
    environment = environment,
    clientKey = clientKey,
    ..
) {
    ..
    instantPayment(PaymentMethodTypes.TWINT) {
        setActionHandlingMethod(ActionHandlingMethod.PREFER_WEB)
    }
}
```
**Note:** With `PREFER_WEB` the redirect flow will be triggered, so you have to also [handle the redirect flow](https://docs.adyen.com/online-payments/build-your-integration/sessions-flow/?platform=Android&integration=Components&version=5.3.1#handle-a-redirect).

Could you please check if this works for you and let me know if you have any questions?

**Comment by ronaldvanduren** [comment]
Hi @araratthehero, thanks for the swift reply! 

I am unable to instantiate the component (but honestly, I also don't know what to expect behaviour-wise from this component). I will wait until the official documentation is released and park this feature for now. (We can still use the web flow until then).

**Comment by araratthehero** [comment]
Hi @ronaldvanduren,

In the next release, the Twint implementation will be updated and the integration will be simplified. We'll keep you informed about these changes.

If you have any questions in the meantime, please contact us.

**Comment by ozgur00** [comment]
Hi @ronaldvanduren, we have released [5.7.0](https://github.com/Adyen/adyen-android/releases/tag/5.7.0) where you can find the documentation for [Twint](https://github.com/Adyen/adyen-android/blob/5.7.0/docs/payment-methods/TWINT.md).
